### ISSUE TITLE
Rendered HTML has duplicate titles, intentional?

### ISSUE THREAD

**Issue by oodler577** [perladvent/Perl-Advent#231]
![image](https://user-images.githubusercontent.com/69679854/204339657-12052724-7891-479d-a986-9b4814a12077.png)

For example; upon looking at previous years Advents, the rendered HTML shows the main title only once. This may be intentional, but seems superfluous and potentially on purpose (for some reason).

**Comment by oalders** [comment]
Are you seeing this issue at all on https://staging.perladvent.org/?

**Comment by oodler577** [comment]
> Are you seeing this issue at all on https://staging.perladvent.org/?

No, but just so I am looking at the same stuff you're talking about; the only Advent calendars and entries I can get to from that link are from 2021 and before.

**Comment by oalders** [comment]
Right, the 2022 archives should start to exist on Dec 1.

**Comment by oalders** [comment]
But that site was just generated earlier this month using the same generation tools that we'll be using in 2022.

**Comment by oalders** [comment]
So, I'm wondering if the issue is the inclusion of the first of the `=head1 NAME` in a418eb19c0d62022b9, since that seems to be auto-created from the article metadata.

**Comment by oodler577** [comment]
Seems so - I'll stand by on my article PRs if I should adjust it to not include the title twice (both do bc this is how the others I looked at were).

**Comment by oalders** [comment]
I think we can do a general cleanup of the titles after merge rather than forcing everyone to update their PRs, so let's not worry about that being a blocker.

**Comment by oodler577** [comment]
* Olaf Alders ***@***.***> [2022-11-28 14:42:40 -0800]:

> I think we can do a general cleanup of the titles after merge rather than forcing everyone to update their PRs, so let's not worry about that being a blocker.

Works for me!

> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/perladvent/Perl-Advent/issues/231#issuecomment-1329849191
> You are receiving this because you authored the thread.
> 
> Message ID: ***@***.***>

-- 
--
***@***.***
***@***.***
SDF-EU Public Access UNIX System - http://sdfeu.org
irc.perl.org #openmp #pdl #native
